src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java

```diff
@@ -173,4 +173,7 @@ public class PrlAppsConstants {
     public static final String CITIZEN_ROLE = "citizen";
 
     public static final String OTHER = "Other";
+
+    public static final String SERVICE_ID = "ABA5";
+    public static final String FAMILY_COURT_TYPE_ID = "18";
```

Comment:
why we have defined FAMILY_COURT_TYPE_ID 18 only, is this a specific configuration?

Reply:
yes for family courts